### PR TITLE
Jormun: Add Timeo Integration Tests

### PR DESCRIPTION
**Pull request by xlqian**
http://jira.canaltp.fr/browse/NAVP-459

### CODE REVIEW

**Comment by xlqian** [comment]
Jenkins,  test this please!

File: source/jormungandr/jormungandr/realtime_schedule/timeo.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def __init__(self, id, service_url, service_args, timezone, timeout=10):
         self.service_url = service_url
         self.service_args = service_args
         self.timeout = timeout  # timeout in seconds
-        self.id = id
+        self.service_id = id
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/realtime_schedule/timeo.py]: I don't like service, there is a lot of things already called "service" in the transport universe

Comment (xlqian) [inline source/jormungandr/jormungandr/realtime_schedule/timeo.py]: rt_system_id?

Comment (xlqian) [inline source/jormungandr/jormungandr/realtime_schedule/timeo.py]: proxy_id?

**Pull request merged**